doc/index.rst

```diff
@@ -0,0 +1,600 @@
+
+Welcome to typing_extensions's documentation!
+=============================================
+
+``typing_extensions`` provides common building blocks for type annotations
+that are understood by static type checkers.
+
+The ``typing_extensions`` module serves two related purposes:
```

Comment:
It would be nice to mention (and link to) the `typing` module in the first paragraph, I think. Maybe something like:

```suggestion
``typing_extensions`` complements the standard-library :py:mod:`typing` module,
providing runtime support for type hints as specified by PEP 484 and subsequent
PEPs. The module serves two related purposes:
```

Reply:
It would be nice to mention (and link to) the `typing` module in the first paragraph, I think. Maybe something like:
Also add a link:
```suggestion
``typing_extensions`` complements the standard-library :py:mod:`typing` module,
providing runtime support for type hints as specified by :pep:`484` and subsequent
PEPs. The module serves two related purposes:
```